### ISSUE TITLE
Consider other models

### ISSUE THREAD

**Issue by JoshVarty** [JoshVarty/AudioTagging#21]
Probably should have started out with this, but we should compare the other models.

### ResNet-18
```
97	0.035250	0.029217	0.664344	00:39
98	0.035705	0.029428	0.662940	00:39
99	0.035190	0.029455	0.662831	00:39
```

### ResNet-50

```
97	0.043572	0.028318	0.665870	00:43
98	0.043541	0.027721	0.672451	00:43
99	0.043261	0.027860	0.665838	00:43
```

### VGG-16 BN

```
97	0.041682	0.025638	0.700495	01:03
98	0.042080	0.025700	0.704907	01:03
99	0.041788	0.025718	0.704059	01:03
```

### VGG-19 BN
```
97	0.045181	0.027131	0.674292	01:12
98	0.044569	0.027153	0.676560	01:12
99	0.044443	0.027219	0.676386	01:12
```

**Comment by JoshVarty** [comment]
## Conv 64-128-256-512

```
97	0.086184	0.028476	0.678350	00:52
98	0.085402	0.028466	0.679489	00:52
99	0.085432	0.028146	0.682341	00:52
```

**Comment by JoshVarty** [comment]
## xresnet50
```
97	0.078883	0.025930	0.718647	00:51
98	0.078497	0.026046	0.718951	00:49
99	0.078972	0.025905	0.716066	00:51
```
Seems to be a bit better

**Comment by JoshVarty** [comment]
## xresnet18

```
97	0.074626	0.028025	0.690300	00:36
98	0.075188	0.027592	0.690504	00:36
99	0.074820	0.027749	0.687696	00:36
```

**Comment by JoshVarty** [comment]
## xresnet101

```
97	0.080427	0.025303	0.717841	01:13
98	0.079735	0.025446	0.712157	01:13
99	0.080249	0.025226	0.715997	01:13
```

*I should mention that there were a few iterations in which it reached  an f-score `>0.72`

**Comment by JoshVarty** [comment]
## xresnet101 with stacked random crops

```
97	0.076211	0.026887	0.702940	01:15
98	0.076367	0.026963	0.685226	01:16
99	0.076069	0.027496	0.682742	01:14
```

**Comment by JoshVarty** [comment]
## xresnet101 with stacked sequential crops 

```
97	0.074217	0.027665	0.680261	01:16
98	0.074828	0.027505	0.690851	01:14
99	0.074364	0.027654	0.696997	01:12
```

**Comment by JoshVarty** [comment]
## xresnet101 with 256x256 images*

```
97	0.084554	0.342670	0.635889	04:06
98	0.084848	0.505083	0.661808	04:05
99	0.084444	0.454247	0.670786	04:05
```

*One thing I noticed is that validation loss is still fluctuating. This may be a sign that we just have to train for more epochs.
